static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -482,10 +482,25 @@ export function SourceMapsDebuggerModal({
       <Body>
         <p>
           {t(
-            "It looks like the original source code for this stack frame couldn't be determined when this error was captured. To get the original code for this stack frame, Sentry needs source maps to be configured."
+            'It looks like the original source code for this stack frame. To get the original code for this stack frame, Sentry needs source maps to be uploaded.'
```

Comment:
I believe you removed part of the first sentence by mistake
```suggestion
            'It looks like the original source code for this stack frame couldn't be determined when this error was captured. To get the original code for this stack frame, Sentry needs source maps to be uploaded.'
```